Feliz/Properties.fs

```diff
@@ -911,20 +1331,78 @@ type prop =
+    static member inline pointsAtY (value: float) = Interop.mkAttr "pointsAtY" value
+    /// Represents the y location in the coordinate system established by attribute primitiveUnits 
+    /// on the <filter> element of the point at which the light source is pointing.
+    static member inline pointsAtY (value: int) = Interop.mkAttr "pointsAtY" value
+
+    /// Represents the y location in the coordinate system established by attribute primitiveUnits 
+    /// on the <filter> element of the point at which the light source is pointing, assuming that, 
+    /// in the initial local coordinate system, the positive z-axis comes out towards the person 
+    /// viewing the content and assuming that one unit along the z-axis equals one unit in x and y.
+    static member inline pointsAtZ (value: float) = Interop.mkAttr "pointsAtZ" value
+    /// Represents the y location in the coordinate system established by attribute primitiveUnits 
+    /// on the <filter> element of the point at which the light source is pointing, assuming that, 
+    /// in the initial local coordinate system, the positive z-axis comes out towards the person 
+    /// viewing the content and assuming that one unit along the z-axis equals one unit in x and y.
+    static member inline pointsAtZ (value: int) = Interop.mkAttr "pointsAtZ" value
+
+    /// Indicates how a <feConvolveMatrix> element handles alpha transparency.
+    static member inline preserveAlpha (value: bool) = Interop.mkAttr "preserveAlpha" value
 
     /// A URL for an image to be shown while the video is downloading. If this attribute isn't specified, nothing
     /// is displayed until the first frame is available, then the first frame is shown as the poster frame.
     static member inline poster (value: string) = Interop.mkAttr "poster" value
 
-    /// SVG attribute to define the radius of a circle
+    /// SVG attribute to define the radius of a circle.
     static member inline r (value: float) = Interop.mkAttr "r" value
-    /// SVG attribute to define the radius of a circle
+    /// SVG attribute to define the radius of a circle.
     static member inline r (value: ICssUnit) = Interop.mkAttr "r" value
-    /// SVG attribute to define the radius of a circle
+    /// SVG attribute to define the radius of a circle.
     static member inline r (value: int) = Interop.mkAttr "r" value
 
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (value: float) = Interop.mkAttr "radius" value
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (value: int) = Interop.mkAttr "radius" value
+    /// Represents the radius (or radii) for the operation on a given <feMorphology> filter primitive.
+    static member inline radius (xRadius: float, yRadius: float) = Interop.mkAttr "radius" (xRadius, yRadius)
```

Comment:
When the `radius` property takes two numbers as input, more specifically as [Number-optional-number](https://developer.mozilla.org/en-US/docs/Web/SVG/Content_type#Number-optional-number) the the values should be serialized to a comma-separated string:
```fs
static member inline radius (xRadius: float, yRadius: float) = 
    Interop.mkAttr "radius" (unbox<string> xRadius  + "," + unbox<string> yRadius)
```